### ISSUE TITLE
Wa-Page / Footer is offscreen

### ISSUE THREAD

**Issue by GedMarc** [shoelace-style/webawesome-alpha#127]
### Describe the bug
When using only a few parts on wa-page, the footer is offscrenn

### To Reproduce

```
<wa-page>
	<div slot="banner">
		<p>This is the banner</p>
	</div>
	<nav slot="navigation">
		<a href="#contentMain">This is the navigation</a>
	</nav>
	<main class="wa-body-l">

		<h2 id="contentMain"></h2>
		<div>
			<p>main layout</p>
		</div>
	</main>
	<div slot="footer">
		<p>This is the footer</p>
	</div>
</wa-page>
```

### Screenshots

![image](https://github.com/user-attachments/assets/102e4d13-d237-4701-81c4-36f0ed41884a)

![image](https://github.com/user-attachments/assets/5d1b0094-3b05-4af4-9b93-fecf2fd30b4d)

**Comment by KonnorRogers** [comment]
While this isn't technically a bug, it's currently expected behavior. The problem is mainly around when you have content overflowing in the "main" section and then you scroll, if you allow space for the footer on screens that don't scroll you hit weird territory where there's extra white space at the bottom of the menu / aside regions.

I *think* as I'm looking at this problem again we may be able to fix this, but it does require some calculation around knowing if the main region is going to overflow. And if it's not overflowing, then we can safely reduce the height of the aside / menu regions.

TLDR: we're aware, I think I know of some possible fixes, but for now this is expected behavior, although possibly not desirable behavior.

**Comment by GedMarc** [comment]
hmm, positioning the slot footer bottom:0 is a possible option, then calculating the main can stay as default :)
